# config file for replaying trajectories

# control settings
v: 500
z: z100
mode: 'states' # can also be poses

# playback settings
target_fps: 20
subsample: 1

# data settings
data_path: /mnt/hdd/data/yumi_flexpendant/ #/mnt/hdd/data/yumi_teleop/, /mnt/hdd/data/yumi_kinesthetics
